### PR TITLE
Add bio, hireable, and twitter_username fields to Person

### PR DESCRIPTION
**Pull request by Chew**
# Description 
** Describe your change here** 
This adds 3 fields to the GHPerson abstract method
`getBio()` which returns the user's bio as a String
`getTwitterUsername()` which returns their Twitter username as a String
`isHireable()` which returns if they're hirable as a boolean. (The API itself says "hireable", I am just going off of what that says)

These all appear for People and Organizations as well.

# Before submitting a PR:
We love getting PRs, but we hate asking people for the same basic changes every time. 

- [x] Push your changes to a branch other than `master`. Create your PR from that branch.    
- [x] Add JavaDocs and other comments
- [x] Write tests that run and pass in CI. See [CONTRIBUTING.md](CONTRIBUTING.md) for details on how to capture snapshot data.
- [x] Run `mvn clean compile` locally. This may reformat your code, commit those changes.
- [x] Run `mvn -D enable-ci clean install site` locally. If this command doesn't succeed, your change will not pass CI.

### CODE REVIEW

**Comment by Chew** [comment]
Oh I see, if I do /users/:org it shows up, but if I do /orgs/:org it doesn't show up. Why is the API so confusing? I'll move it over!

File: src/test/java/org/kohsuke/github/GHUserTest.java [hunk 1]
```diff
@@ -102,4 +102,11 @@ public void createAndCountPrivateRepos() throws IOException {
             repository.delete();
         }
     }
+
+    @Test
+    public void verifyBioAndHireable() throws IOException {
+        GHUser u = gitHub.getUser("GitHub");
+        assertNotNull(u.getBio());
+        assertNotNull(u.isHireable());
```
Comment (bitwiseman) [inline src/test/java/org/kohsuke/github/GHUserTest.java:110]: `boolean` is never `null`.  Please find a user where it is `true`.

File: src/test/java/org/kohsuke/github/GHUserTest.java [hunk 2]
```diff
@@ -102,4 +102,11 @@ public void createAndCountPrivateRepos() throws IOException {
             repository.delete();
         }
     }
+
+    @Test
+    public void verifyBioAndHireable() throws IOException {
+        GHUser u = gitHub.getUser("GitHub");
+        assertNotNull(u.getBio());
```
Comment (bitwiseman) [inline src/test/java/org/kohsuke/github/GHUserTest.java:109]: Change this to an `assertThat()` the checks the value not just not null.

**Comment by bitwiseman** [comment]
@Chew 
And finally a user or org with non-null `twitter_username`.

**Comment by Chew** [comment]
I changed it to my profile since it
1) had a not null bio
2) had a not null twitter username
3) hireable is true

**Pull request merged**